opm/output/eclipse/Summary.cpp

```diff
@@ -940,6 +945,9 @@ Summary::Summary( const EclipseState& st,
 	    this->handlers->handlers.emplace_back( nodeptr, handle );
 	}
     }
+    for ( auto keyword : unsupported_keywords ) {
```

Comment:
As suggested I think the log call could come directly; if you prefer to assemble the set the iteration should be:
```C++
for (const auto& keyword: ...
```

Reply:
Of course, I'll fix this. I'm trying to improve and get rid of those annoying obvious mistakes..